source/Willow-Tests/CombinedWebInteractionInterpreterTest.class.st

```diff
@@ -828,7 +831,7 @@ CombinedWebInteractionInterpreterTest >> testToggleCssClass [
 
 	interpreter := CombinedWebInteractionInterpreter combiningInterpretersOf: self buttons.
 
-	interpreter toggleCssClass: WillowCssStyles loading.
+	interpreter toggleCssClass: Classification >> #loading.
```

Comment:
WillowCSSClasses instead of Classification? I believe Classification is too generic

Reply:
Classes is too confusing, CSS is incorrect since they are technically part of HTML. In order to reference the namespace, you must explicitly define access to it in the pools of the class definition.